phpseclib/Crypt/Blowfish.php

```diff
@@ -605,11 +605,11 @@ private static function expandstate(array $data, array $key, array &$sbox0, arra
             $p[14] ^ $key[14],
             $p[15] ^ $key[15],
             $p[16] ^ $key[0],
-            $p[17] ^ $key[1]
+            $p[17] ^ $key[1],
         ];
 
         // @codingStandardsIgnoreStart
-        [ $p[0],  $p[1]] = self::encryptBlockHelper($data[ 0]         , $data[ 1]         , $sbox0, $sbox1, $sbox2, $sbox3, $p);
+        [ $p[0],  $p[1]] = self::encryptBlockHelper($data[ 0], $data[ 1], $sbox0, $sbox1, $sbox2, $sbox3, $p);
```

Comment:
Same thing as above.

Reply:
Using loop so we don't have to use special styling